brainreg_segment/segment.py

```diff
@@ -414,9 +414,15 @@ def initialise_loaded_data(self):
         else:
             self.hemispheres_data = self.viewer.layers["Hemispheres"].data
 
+        self.prevent_layer_edit()
+
         self.initialise_segmentation_interface()
         self.status_label.setText("Ready")
 
+    def prevent_layer_edit(self):
+        for layer in self.viewer.layers:
+            layer.editable = False
```

Comment:
Will this lock all the layers a user has loaded before loading anything with `brainreg-segment`? If so isn't that quite blunt, and would it be possible to just set `editable` on the layers as they're loaded above?

Reply:
Ah yes good point.

Reply:
I've addressed this. Only layers added by the widget are set to non-editable. I thought I could just `add_image(data, editable=False)` but that's not supported.